src/app/core/header/header.component.html

```diff
@@ -4,9 +4,15 @@
 
   <div class="filler"></div>
 
-  <!-- Place info about current filters here! -->
+  <div class="filter-labels">
+    <div>Sex: {{currentFilters.sex[0]}}</div>
+    <div>Age: {{currentFilters.ageRange[0]}}-{{currentFilters.ageRange[1]}}</div>
+    <div>BMI: {{currentFilters.BMIRange[0]}}-{{currentFilters.BMIRange[1]}}</div>
+  </div>
 
-  <mat-icon class="download" aria-hidden="false" aria-label="Download content" (click)="downloadClicked.emit()">
-    cloud_download</mat-icon>
-  <ccf-info-button></ccf-info-button>
+  <a href="https://sampledata.hubmapconsortium.org/home">
```

Comment:
Add `rel="noreferrer noopener" target="_blank"` to the a tag (so that it opens in a new window)